src/fs/copy.go

```diff
@@ -64,27 +64,26 @@ func RecursiveCopyOrLinkFile(from string, to string, mode os.FileMode, link, fal
 				return os.MkdirAll(dest, DirPermissions)
 			}
 			if fileMode.IsSymlink() {
-				return recursiveCopyOrLinkSymlink(name, dest, mode, link, fallback)
+				return copySymlink(name, dest)
 			}
 			return CopyOrLinkFile(name, dest, fileMode.ModeType(), mode, link, fallback)
 		})
 	}
 	return CopyOrLinkFile(from, to, info.Mode(), mode, link, fallback)
 }
 
-// recursiveCopyOrLinkSymlink will resolve the symlink, and recursively copy the content to the destination
-func recursiveCopyOrLinkSymlink(name, dest string, mode os.FileMode, link, fallback bool) error {
+// copySymlink will resolve the symlink, and recursively copy the content to the destination
+func copySymlink(name, dest string) error {
 	resolvedPath, err := os.Readlink(name)
 	if err != nil {
 		return err
 	}
-	if !filepath.IsAbs(resolvedPath) {
-		resolvedPath = filepath.Join(filepath.Dir(name), resolvedPath)
-	}
-	if err := RecursiveCopyOrLinkFile(resolvedPath, dest, mode, link, fallback); err != nil {
-		return err
+
+	if resolvedPath == "../mocha/bin/_mocha" {
```

Comment:
Is this supposed to be here?

Reply:
Yes this is absolutely critical to the operation of the buld system ;P